cms/djangoapps/modulestore_migrator/models.py

```diff
@@ -145,6 +145,10 @@ class ModulestoreMigration(models.Model):
             "is the migration failed?"
         ),
     )
+    migration_summary = models.JSONField(
+        default=dict,
+        help_text=_("Summary that contains number of sections, subsections, units and components migrated"),
+    )
```

Comment:
@ChrisChV Do we need this new field? All data can be fetched from [BlockMigrationInfo](https://github.com/openedx/edx-platform/blob/aef9e53dcf4bd2470eb3adb7bef6a792dbfc9769/cms/djangoapps/modulestore_migrator/rest_api/v1/views.py#L502) api. We can fetch all blocks for a given target_key and source_key, then calculate the number of different blocks in the frontend itself.